File: examples/miscellaneous/backgroundTypes/backgroundFunction.m
function background = backgroundFunction(xdata,params)

% Split up the params array....
Ao = params(1);
k = params(2);
backConst = params(3);

% Make an exponential decay background....
background = zeros(numel(xdata),1);
for i = 1:numel(xdata)
    background(i) = Ao*exp(-k*xdata(i)) + backConst;
end


end

File: examples/Miscellaneous/backgroundTypes/backgroundFunction.py
import numpy as np

def backgroundFunction(xdata, params):

    # Split up the params array
    Ao = params[0]
    k = params[1]
    back_const = params[2]

    # Make an exponential decay background
    background = np.zeros(len(xdata))
    for i in range(0, len(xdata)):
        background[i] = Ao*np.exp(-k*xdata[i]) + back_const

    return background
